subiquity/ui/views/ubuntu_advantage.py

```diff
@@ -207,14 +207,21 @@ def __init__(self, parent: UbuntuAdvantageView, services: list):
         ok = ok_btn(label=_("OK"), on_press=self.ok)
 
         title = _("Available Services")
-        header = _("List of services that will be enabled through your "
+        header = _("List of services that are available through your "
                    "Ubuntu Advantage subscription:")
 
-        widgets: List[Widget] = [Text(header)]
-        widgets.extend([Text(f"* {svc['description']}") for svc in services])
-        widgets.append(button_pile([ok]))
+        widgets: List[Widget] = [
+            Text(header),
+            Text(""),
+            Pile([Text(f"* {svc['description']}") for svc in services]),
+            Text(""),
+            Text("You can enable these services using the `ua` command-line "
+                 "tool."),
```

Comment:
Though it's important that users wait until the install is complete and not do so during install time.  I would like you to include some wording to that effect.

Reply:
Reworded to:
"Once the installation has finished, you can enable these services using the `ua` command-line tool."

is it better?

Reply:
Yes, thanks